library/std/src/sys/unix/ext/process.rs

```diff
@@ -25,6 +25,11 @@ pub trait CommandExt {
     #[stable(feature = "rust1", since = "1.0.0")]
     fn gid(&mut self, id: u32) -> &mut process::Command;
 
+    /// Sets the supplementary group IDs for the calling process. Translates to
+    /// a `setgroups` call in the child process.
+    #[unstable(feature = "setgroups", issue = "38527", reason = "")]
+    fn groups(&mut self, groups: Vec<u32>) -> &mut process::Command;
```

Comment:
Hmm, `Vec` seems like a bit of an arbitrary choice here, because what we want is just a set of ids, not really a container we can push/pop.

Since either the caller or the implementation will need to allocate this `Vec` either way I think I'd prefer the implementation do it, and make the signature:

```rust
fn groups(&mut self, groups: &[u32]) -> &mut process::Command;
```

Reply:
The Box::new([]) seems kind of awkward to me. Let me know if that is incorrect.